target/linux/ath79/image/generic.mk

```diff
@@ -1236,6 +1236,16 @@ define Device/glinet_gl-usb150
 endef
 TARGET_DEVICES += glinet_gl-usb150
 
+define Device/glinet_gl-x300b
+  SOC := qca9531
+  DEVICE_VENDOR := GL.iNet
+  DEVICE_MODEL := GL-X300B
+  DEVICE_PACKAGES := kmod-usb2 block-mount
+  IMAGE_SIZE := 16000k
+  SUPPORTED_DEVICES += gl-x300b glinet,gl-x300b
```

Comment:
The second one is already added automatically. Why would you need the first one?